reputation/microeconomic_reputation.tex

```diff
@@ -0,0 +1,367 @@
+    accept it for the sake of the brevity of this document. Also "truth about Golem is known" is an ideal world we would like to live in.
+
+\item The expected utility and money are interchangeable, that is - for every agent $A$ and a pair of decisions $(D_1, D_2): E_A(D_1) < E_A(D_2)$, there is an amount of money X that
+    can be given to the agent so that $E_A(D_1) + U_A(X) = E_A(D_2)$. This way we can treat utilities as if they were money, and thus compare them between different agents.
+
+\item Lets define the Golem Value as:
+
+\begin{equation}
+    V_G = \sum_{A \in all\_agents\_using\_Golem}E_A(USE\_GOLEM)
+\end{equation}
+(Note that - because of assumption 1 - $E_A(USE\_GOLEM)$ is always positive, decision "use Golem" is only made by agents who expect to profit from it).
+
+We assume that maximization of $V_G$ is one of the goals of the Golem Factory - and from the point of view of this document, the only goal.
+\end{enumerate}
+
+\section{The "reputation" and its purpose}
+\subsection{Notation}
+
+We'll be using a bunch of different symbols, but they follow a clear common pattern, so please don't be frightened.
+
+\begin{itemize}
+\item $V_A(a)$ is the total value of the agreement $a$ from the POV of agent $A$. As per assumption 3, this is "utility expressed as money".\footnote{
+I.e. $V_A(a) = X$ means "when agent $A$ takes part in the agreement $a$, their hapiness changes as if they were given $X$ money".}, i.e.:
+    \begin{itemize}
+        \item For the provider it is the utility of the money received decreased by the utility cost of the hardware/electricity/etc.
+        \item For the requestor it is the utility of the resources obtained decreased by the utility cost of the money spent
+    \end{itemize}
+\item $V_P(a)$/$V_R(a)$ are agreement values from the POV of (respectively) provider/requestor.
+\item $V_{PN}(a)$/$V_{RN}(a)$ are nominal (i.e. negotiated) values of the agreement\footnote{
+   Note that this doesn't translate directly to the agreement amount. E.g. when agreed amount is positive but so low it doesn't cover provider's costs the $V_{PN}$ is negative.
+}. They equal $V_P(a)$/$V_R(a)$ if neither side breached the agreement $a$.
+\item $V_{AL}(a)$ is the value lost by agent $A$ because of the other side breaching the agreement $a$.
+\item $V_{AG}(a)$ is the value gained by agent $A$ when they break the agreement $a$.
+\item $V_{PL}$, $V_{RL}$, $V_{PG}$, $V_{RG}$ are $V_{AL}$/$V_{AG}$ from the POV of provider/requestor.
+\item $C_A$ is the cost of the participation in the Golem market for agent $A$ that is not related to any particular agreement 
+(e.g. the opportunity cost\footnote{\href{https://en.wikipedia.org/wiki/Opportunity\_cost}{https://en.wikipedia.org/wiki/Opportunity\_cost}} of the hardware offered on the market, 
+or the cost of writing the requestor agent).
+\end{itemize}
+
+\subsection{Dissolving the Golem Value}
```

Comment:
"Dissolving"? surely you don't mean that ;)

Reply:
Hmm, from where I come from it's common to say e.g. "to dissolve the question" with a meaning "to get rid of all terminology-related confusion & understand what the question is really about". So I meant something similar, but maybe this is not how others understand this word :/
Anyway, just tell me to change this, and I will :)

Reply:
I meant it has some negative connotations as well (maybe even mostly), at least according to Webster:

a : to cause to disperse or disappear : destroy 
b : to separate into component parts : disintegrate 
c : to bring to an end : terminate

Reply:
OK, that makes sense. I'll change this.